addons/headless/CfgEden.hpp

```diff
@@ -0,0 +1,35 @@
+class Cfg3DEN {
+    class Object {
+        class AttributeCategories {
+            class ace_attributes {
+                class Attributes {
+                    class GVAR(blacklist) {
+                        displayName = CSTRING(BlacklistEden);
+                        tooltip = CSTRING(BlacklistEdenDesc);
+                        property = QGVAR(blacklist);
+                        control = "Checkbox";
+                        condition = "objectVehicle";
```

Comment:
I am still not too happy about this. BI wiki says:
> objectVehicle - Vehicle which can be boarded

But this attribute should work only on units directly (and groups). Going by the wiki, `objectControllable` seems like a better condition.